### PR TITLE
docs: reorganise documentation with different main sections

### PR DESCRIPTION
**Pull request by alban**
# docs: reorganise documentation

The documentation now has the following main sections:

- getting started: installation instructions split by target
- core concepts
- development
- guides
- built-in gadgets

## How to use

```
make website-local-update
```

## Testing done

<details>

image

</details>

### CODE REVIEW

File: docs/devel/how-to-write-a-gadget.md
Comment (mauriciovasquezbernal) [inline docs/devel/how-to-write-a-gadget.md:1]: Perhaps we can remove this file right away?

File: docs/guides/run.md
Comment (mauriciovasquezbernal) [inline docs/guides/run.md:1]: Change the title to just "running a gadget"?

File: docs/guides/_index.md
Comment (mauriciovasquezbernal) [inline docs/guides/_index.md:1]: Any reason not to have script and traceloop here too?

**Review by mauriciovasquezbernal — COMMENTED** [review]
I think it goes into the right direction. Just a couple of comments.

File: docs/guides/_index.md
Comment (alban) [reply docs/guides/_index.md:1]: Since they are built-in gadgets, they are in the builtin-gadgets section.

Comment (mauriciovasquezbernal) [reply docs/guides/_index.md:1]: I don't see the built-in gadgets section while testing locally. Did you forgot `git add docs/builtin-gadgets/_index.md`?

Comment (alban) [reply docs/guides/_index.md:1]: You're right, I forgot it. It's added now.

**Review by mauriciovasquezbernal — APPROVED** [review]
LGTM. I think it's a step on the right direction. Thanks for taking care of this.

**Pull request merged**